test.php

```diff
@@ -0,0 +1,113 @@
+<?php
```

Comment:
This was just a quick test file to test what the performance decrease/impact would be. Since the premise of being able to style cells would seem te be a lot heavier